requirements/examples.txt

```diff
@@ -1,10 +1,11 @@
-boto3 ~= 1.17
-ipyplot ~= 1.1
-ipywidgets ~= 7.6
-jupyterlab ~= 3.0
-matplotlib ~= 3.3
-pystac ~= 1.7.3
-rasterio ~= 1.2
+boto3 ~= 1.28
 intake-stac ~= 0.4.0
-netCDF4
-numpy
+ipyplot ~= 1.1
+ipywidgets ~= 8.1
+jupyterlab ~= 4.0
+matplotlib ~= 3.8
+netCDF4 ~= 1.6
+numpy ~= 1.26
+pillow ~= 10.1  # A dependency of ipyplot, pinned to avoid critical vulnerability.
```

Comment:
I'm guessing future releases of `ipyplot` will bump their own required version of `pillow`, so this line can be removed at a later date. 🤞